sdks/python/container/build.gradle

```diff
@@ -22,10 +22,9 @@ applyDockerNature()
 
 description = "Apache Beam :: SDKs :: Python :: Container"
 
-// Figure out why the golang plugin does not add a build dependency between projects.
-// Without the line below, we get spurious errors about not being able to resolve
-// "./github.com/apache/beam/sdks/go"
-resolveBuildDependencies.dependsOn ":beam-sdks-go:build"
```

Comment:
Just to confirm: we don't need this anymore? I think it's also present in the other container build files.

Reply:
I did not get a build error without it. Is it valid to assume that when the build is successful it's no longer needed or are the symptoms you observed more subtle?

Reply:
Not sure. I think @lukecwik might be the one how added it originally. Luke?

Reply:
The gogradle plugin had (or still has) issues within multiproject builds where it would not correctly add transitive dependencies of projects to the current build.

This may have been fixed since there have been updates to the gogradle plugin since last year. Try deleting all instances of resolveBuildDependencies.dependsOn ":beam-sdks-go:build" in all the build files and trying a clean build. If it works we could remove this here and elsewhere.

Reply:
I'm adding back the explicit dependency for this PR and will take up removing all occurrences in a separate PR.